docs/use-case-descriptions/message-validation.md

```diff
@@ -13,25 +13,73 @@ If it fails validation, a response message is sent with ‘rejected’ in the re
-| Reference Period mismatch        | The message(s) with the sequence where is referred to contains a different Period.                                                                                                                                    |
-| Reference message expired        | The message that is referred to is expired.                                                                                                                                                                           |
-| Reference message revoked        | The message that is referred to is revoked.                                                                                                                                                                           |
-| ISPs out of bounds               | One or more ISPs are outside the tolerated boundaries: ISPs do not exist.                                                                                                                                             |
-| ISP conflict                     | One or more ISPs are defined more than once, possibly because of an incorrect duration.                                                                                                                               |
-| Period out of bounds             | Period of the message is inappropriate.</br>For example: a FlexRequest with a Period in the past or a settlement item in a FlexSettlement with a Period outside the concerning settlement period.                     |
-| ExpirationDateTime out of bounds | ExpirationDateTime is in the past or exceeds the ISPs in the message.                                                                                                                                                 |
+
+| RejectionReason | Cause of rejection | Applicable to messages |
+|---|---|---|
+| Unknown SenderDomain | SenderDomain specified in the SignedMessage is unknown to the receiving implementation. | SM |
+| Invalid SenderRole | The SenderRole specified in the SignedMessage is invalid. | SM |
+| Invalid Sender combination | The combination SenderDomain and SenderRole in the SignedMessage is invalid.| SM |
+| Mismatch SenderDomain | SenderDomain specified in the SignedMessage doesn't match with the SenderDomain in the inner XML message. | SM & All other message types |
+| Invalid SignedMessage | Despite being schema-compliant, the syntax, type or semantics of the SignedMessage were unacceptable for the receiving implementation. | SM |
+| Invalid Message | Despite being schema-compliant, the syntax, type or semantics of the message were unacceptable for the receiving implementation. | All message types |
+| Unknown RecipientDomain | The RecipientDomain specified in the inner XML message is not known to the receiving implementation. | All message types |
+| Unknown SenderDomain | SignedMessage\SenderDomain doesn't match SignedMessage\Body\SenderDomain. | SM & All other message types |
```

Comment:
This  was already defined on line 22

Reply:
That's correct. I wanted to distinguish between Unknown SenderDomain in the SignedMessage and in the other message types. That way when you receive a validation error you can clearly see where it goes wrong. The description was incorrect.
How do you see this?

Reply:
it's fine like this; but a minor thing: "_SignedMessage\SenderDomain doesn't match SignedMessage\Body\SenderDomain._" => it makes more sense to use forward slashes here